.github/workflows/deepseek-cr2.yaml

```
@@ -0,0 +1,97 @@
+            ## 优化建议
+            ### 建议 1:
+            - 建议内容：xxxx
+            - 代码示例：```代码```
+            ## 总结
+            - `xxx/xxx/test_error_code1.go`: 
+              - 问题点：
+              - 优化点：
+              - 修复建议：
+              - 总结：
+          user-prompt: "请审查以下 Go 文件的代码变更"
+          exclude-patterns: ${{ lzb.AI_REVIEW_EXCLUDE_PATTERNS }}
```

actionlint (1.7.4)

undefined variable "lzb". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)